contrib/cmd/sd-helper/helper.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/cgroups/systemd"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+// version will be populated by the Makefile, read from
+// VERSION file of the source code.
+var version = ""
+
+// gitCommit will be the hash that the binary was built from
+// and will be populated by the Makefile.
+var gitCommit = ""
```

Comment:
Will this utility only be used for tests? Wondering if adding version information is a bit too much for that (or even using `urfave/cli` for this).

Reason I'm looking at the `version` variable, is also that if we want to switch to `//go:embed` (https://github.com/opencontainers/runc/pull/3163), the path to files to embed cannot traverse to parent directories (so `//go:embed ../../VERSION` won't work) 😅

Reply:
I just copied this bit (as well as using `urlfave/cli`) from `recvtty`.

Let me try to drop both.

Reply:
Fixed, PTAL @thaJeztah